### PR TITLE
Forceverification

### PR DESCRIPTION
**Pull request by originalblend337**

### CODE REVIEW

File: frontends/web/src/routes/account/receive/receive.jsx [hunk 1]
```diff
@@ -142,36 +161,77 @@ export default class Receive extends Component {
...
+                            </a>
                         )
                     }
                     <p class={style.label}>{t('receive.label')} { receiveAddresses.length > 1 ? `(${activeIndex + 1}/${receiveAddresses.length})` : ''}</p>
                     {
                         receiveAddresses.length > 1 && (
-                            <Button
-                                transparent
+                            <a
+                                href="!#"
```
Comment (benma) [inline frontends/web/src/routes/account/receive/receive.jsx]: what is `!#`? (I've only seen `#` before)

File: frontends/web/src/components/copy/Copy.css [hunk 2]
```diff
@@ -1,49 +1,52 @@
 .button {
```
Comment (benma) [inline frontends/web/src/components/copy/Copy.css]: why did you change the style and position of the copy button? tbh I liked the previous version much better, it looked more polished.

File: frontends/web/src/routes/account/receive/receive.jsx (hunk 1 above)
Comment (originalblend337) [reply frontends/web/src/routes/account/receive/receive.jsx]: typo.. lol although there is such thing as a hashbang (#!), where it means something different. fixed.

File: frontends/web/src/components/copy/Copy.css (hunk 2 above)
Comment (originalblend337) [reply frontends/web/src/components/copy/Copy.css]: I'm still not certain that one looks better over another, but since other inputs such as the address field in send has the qr code inside the field, thought it may be good to have the same style across the app.

**Comment by benma** [comment]
Nice work!
